protocol/x/subaccounts/types/isolated_perpetual_position_state_transition_test.go

```diff
@@ -0,0 +1,36 @@
+package types_test
+
+import (
+	"testing"
+
+	"github.com/dydxprotocol/v4-chain/protocol/x/subaccounts/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPositionStateTransitionString(t *testing.T) {
```

Comment:
nit: Is this test necessary? Seems pretty trivial logic

Reply:
Not necessary, but could be useful if increase / decrease of position is a future state to be tracked.